radio_beam/tests/test_beams.py

```diff
@@ -91,6 +218,12 @@ def test_indexing():
     assert np.all(beams[mask].major.value == majors[mask].value)
 
 
+# def test_beams_mult_convolution():
+
```

Comment:
maybe add a comment about the intent of this one and why it's commented out?  Is this something we'll add later?

Reply:
Oops. This should have been deleted